tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "isolatedModules": true,
     "jsx": "react",
     "lib": [
-      "es6"
+      "es6", "dom"
```

Comment:
What is this used for?

Reply:
I will remove it for now. I will take note of it. I believe it is used when manipulating dom elements in typescript when I was trying to get the dom manipulation to work.